client/src/redux/donation-saga.js

```diff
@@ -67,10 +73,24 @@ function* postChargeStripeSaga({ payload }) {
   }
 }
 
+function* postChargeStripeCardSaga({ payload }) {
+  try {
+    yield call(postChargeStripeCard, payload);
```

Comment:
This is not throwing an error (from the API). So, I would not expect the `try` to fail.

Reply:
Yes, that is what I was referring to in my previous comment.
Error handling should work as expected now.